spec/models/vote_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+describe Vote do
+  describe "validations" do
+    describe "value validation" do
+      it "only allows -1 or 1 as values" do
+        expect(@vote.save).to eq(true)
```

Comment:
I think here we are missing the declaration of `@vote`. I remember seeing it in our previous chat but it looks like maybe you forgot to check that in :smile:.